### ISSUE TITLE
contacts sync with google not working. once the sync is done the contacts are loaded but after finishing they all disappear. no workaround found

### ISSUE THREAD

**Issue by steinwayer** [ubports/ubuntu-touch#1043]
### Illustrations

- Device: Nexus 5
- Channel:
- Build: 16.04 ota-7

### Steps to reproduce

try to sync google contacts, it will start syncing but qhen finished the contacts will disappear. tried doing jt also from settings/account with no luck

### Expected behavior

contacts should not disappear after sync is finished
### Actual behavior

contacts disappear after sync is finished

**Comment by mateosalta** [comment]
same on pro-5, I had to export from the contacts.google.com website, and use `syncevolution --import /path/to/file/allcontacts.vcf backend=evolution-contacts database=Personal` to import

**Comment by hummlbach** [comment]
Duplicate of #926 i guess

**Comment by UniversalSuperBox** [comment]
Looks like this is a duplicate of #926. Please continue any further discussion there
